security/pkg/stsservice/server/server.go

```diff
@@ -129,8 +129,10 @@ func (s *Server) validateStsRequest(req *http.Request) (security.StsRequestParam
 		return reqParam, errors.New("request is nil")
 	}
 
-	reqDump, _ := httputil.DumpRequest(req, true)
-	stsServerLog.Debugf("Received STS request: %s", string(reqDump))
+	if stsServerLog.DebugEnabled() {
```

Comment:
Since there is no code to invoke stsServerLog.SetOutputLevel(), is the log level of stsServerLog configured through the GUI ControlZ interface using the method on https://istio.io/latest/docs/ops/diagnostic-tools/controlz/? 
Besides the ControlZ interface,  are there other ways to configure the log level of stsServerLog?

Reply:
Yes, command line flag

Reply:
--log_output_level. We have helm values and annotations that set this